### DISCUSSION TITLE
Update (Item.yml)

### DISCUSSION THREAD

**Discussion by Stainded** [diamond-rip/Eden#192] [Ideas]
It allows the item system to be more customizable and open to change making this kind of option a perfect fit and I will leave the example attached to a ".png" file!

![image](https://user-images.githubusercontent.com/128519059/226739639-4f3cd04b-2db8-4749-8a6b-dadc81e57124.png)

**Comment by Glustiksas** [comment]
This idea is extremely interesting, but will he accept it?

**Reply by Stainded** [reply]
I hope so, these are very important things that any plugin that works with hot-bar items should have

**Reply by liexes** [reply]
I give my vote for it to be implemented

**Reply by GoodestEnglish** [reply]
Will be added soon.

**Reply by Stainded** [reply]
@GoodestEnglish, Will you also announce the /follow and /silent system?
that you will soon start working on such a system?

**Reply by GoodestEnglish** [reply]
> @GoodestEnglish, Will you also announce the /follow and /silent system?
> 
> that you will soon start working on such a system?

Yes
